cmd/createcluster.go

```diff
@@ -152,22 +147,38 @@ func runCreateCluster(ctx context.Context, w io.Writer, conf clusterConfig) erro
 		if err != nil {
 			return err
 		}
+
+		defWAddrs := def.WithdrawalAddresses()
+		defFAddrs := def.FeeRecipientAddresses()
+
+		if len(defFAddrs) != 0 {
```

Comment:
I'd stick to "definition overrides all other configuration flags". So It doesn't state "non-zero values in the definition overrides any provided in other flags."